src/backend/efiling-api/jag-efiling-api.yaml

```
@@ -98,13 +121,43 @@ paths:
             format: uuid
       responses:
         "200":
-          description: Config found
+          description: Filing Package Information found
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/GetSubmissionResponse"
+                $ref: "#/components/schemas/FilingPackage"
+        "404":
+          description: Filing Package Information not found
+  "/submission/{id}/document/{filename}":
+    get:
+      summary: Get document by name
+      operationId: GetSubmissionDocument
+      tags:
+        - submission
+      parameters:
+        - name: id
+          in: path
+          description: A submission id
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: filename
+          in: path
+          description: A document name
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Package Information found
```

```suggestion
          description: Filing Package Information found
```

---

or maybe this should actually be:
```suggestion
          description: Document Information found
```